src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write your code here
+const body = document.body;
+const logo = body.querySelector('.logo');
+
+const createDiv = (state) => {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+
+  switch (state) {
+    case 'Success':
```

Comment:
it's better to move 'Success' and 'Error' to the constants as it will make a code cleaner